### PR TITLE
UI : fix the missing special character in TableEntity type column

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes :

- fix the missing special character in TableEntity type column

#
### Type of change :

- [x] Bug fix

#
### Frontend Preview (Screenshots) :
image (5)

image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/EntityTable/EntityTable.component.tsx [hunk 1]
```diff
@@ -428,12 +428,12 @@ const EntityTable = ({
...
           isReadOnly || (dataTypeDisplay.length < 25 && !isReadOnly) ? (
-            lowerCase(dataTypeDisplay)
+            toLower(dataTypeDisplay)
           ) : (
             <Popover
               destroyTooltipOnHide
-              content={lowerCase(dataTypeDisplay)}
-              overlayInnerStyle={{ maxWidth: '420px' }}
+              content={toLower(dataTypeDisplay)}
+              overlayInnerStyle={{ maxWidth: '480px', wordBreak: 'break-word' }}
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/components/EntityTable/EntityTable.component.tsx:436]: why we need wordBreak? have you checked with long string? its not looking good

**Review by ShaileshParmar11 — COMMENTED** [review]
one comment

File: openmetadata-ui/src/main/resources/ui/src/components/EntityTable/EntityTable.component.tsx (hunk 1 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/components/EntityTable/EntityTable.component.tsx:436]: image

Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/components/EntityTable/EntityTable.component.tsx:436]: `raw_product_catalog` from the sample data

Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/EntityTable/EntityTable.component.tsx]: @ShaileshParmar11  Updated!

**Review by ShaileshParmar11 — APPROVED** [review]

**Pull request merged**